superset/cli.py

```diff
@@ -510,7 +510,7 @@ def import_dashboards(path: str, recursive: bool, username: str) -> None:
         default=False,
         help="recursively search the path for yaml files",
     )
-    def import_datasources(path: str, sync: str, recursive: bool) -> None:
+    def import_datasources_yaml(path: str, sync: str, recursive: bool) -> None:
```

Comment:
I'm a little worried about changing method names. It doesn't look like it's used anywhere super critical (although I wonder if the existing unit tests hit it at all--do you know?). This thing is wrapped in an if statement relating to a feature flag. Any chance we're breaking something? @cccs-RyanS  any thoughts?

Reply:
Yeah, I didn't want to do that, however, invoking that original method would always invoke the other method of the same name, and since there was no way of specifying which of the two I wanted to invoke, Ryan recommended changing the name.